platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/model/internal/deterministic/DeterministicTaskSchedulerBuilder.java

```diff
@@ -95,7 +96,9 @@ public TaskScheduler<OUT> build() {
                     case NO_OP -> new NoOpTaskScheduler<>(model, name, type, flushingEnabled, squelchingEnabled);
                 };
 
-        model.registerScheduler(scheduler, hyperlink);
+        if (type != NO_OP) {
```

Comment:
does this imply NO_OP scheduler don't get a link to a doc? Should they have a documentation? I think yes.

Reply:
This is intentional. This causes a no-op scheduler to completely disappear from the wiring diagram, not just the hyperlink.